### PR TITLE
Documentation added and minor fixes

### LINKED ISSUE #80: Fix NgSpice Model tab under kicadToNgspice
When adding data for `NgSpice Model` under `KicadToNgspice` there are input boxes, for each letter, whereas it should be for each word.
Refer-
image

### PR DESCRIPTION
**Pull request by nilshah98**
**PR Details**
- Documentation added folder-wise
- Minor fixes pertaining to pep8 made
- Updated `python` calls to `python3`
- Resolved #80

### CODE REVIEW

File: src/kicadtoNgspice/Convert.py [hunk 1]
```diff
@@ -440,7 +442,10 @@ def addModelParameter(self, schematicInfo):
                             param = key
                             default = 0
                         # Cheking if value is iterable.its for vector
-                        if hasattr(value, '__iter__'):
+                        if (
```
Comment (sunilshetye) [inline src/kicadtoNgspice/Convert.py]: This does not seem okay.

type() check should be before hasattr() check

File: src/kicadtoNgspice/Model.py [hunk 2]
```diff
@@ -62,14 +62,14 @@ def __init__(self, schematicInfo, modelList, clarg1):
                 # print "Key : ",key
                 # print "Value : ",value
                 # Check if value is iterable
-                if hasattr(value, '__iter__'):
+                if hasattr(value, '__iter__') and type(value) is not str:
```
Comment (sunilshetye) [inline src/kicadtoNgspice/Model.py]: type() check should be before hasattr() check

File: src/ngspicetoModelica/ModelicaUI.py [hunk 3]
```diff
@@ -52,7 +52,7 @@ def browseFile(self):
     def callConverter(self):
 
         try:
-            self.cmd1 = "python ../ngspicetoModelica/NgspicetoModelica.py " + \
+            self.cmd1 = "python3 ../ngspicetoModelica/NgspicetoModelica.py" + \
```
Comment (sunilshetye) [inline src/ngspicetoModelica/ModelicaUI.py]:
```suggestion
            self.cmd1 = "python3 ../ngspicetoModelica/NgspicetoModelica.py " + \
```
space missing after .py

**Comment by nilshah98** [comment]
The requested changes have been made, please review @sunilshetye

**Comment by nilshah98** [comment]
Removed the commented code and moved to try catch block, for `src/config/Appconfig.py`

**Pull request merged**